### PR TITLE
Rename requestOptions to searchParams

### PR DESCRIPTION
**Pull request by codedge**
This PR renames the `requestOptions` variable to `searchParams`, see #58.

### CODE REVIEW

**Comment by curquiza** [comment]
Hello again @codedge!
Thanks for your PR, I would like to merge it, but I can't since there are git conflicts. Could you please update your branch and remove them?

Thanks again for your PR, I see you are really involved in this repo right now, this is really helpful!

**Comment by codedge** [comment]
Happy to help! Conflicts are resolved.

File: src/Engine.php [hunk 1]
```diff
@@ -163,15 +163,9 @@ public function search(string $query, string $indexUid, array $searchParams): ar
 
     /**
      * Search the index and returns the number of results.
-     *
-     * @param string $query
-     * @param string $indexUid
-     * @param array  $requestOptions
-     *
-     * @return int
```
Comment (curquiza) [inline src/Engine.php:171]: Is there a reason to remove the comment about params and return?

Comment (codedge) [reply src/Engine.php:171]: The doc block does not provide any helpful information as the method is fully type-hinted.
As some other PR removes all the doc blocks that are there, although the method is fully type-hinted, I already removed it here.

**Review by curquiza — APPROVED** [review]
Thanks

bors merge

**Pull request merged**